build.sbt

```diff
@@ -14,7 +14,7 @@ lazy val commonSettings = Seq(
   organization := "com.cognite.spark.datasource",
   organizationName := "Cognite",
   organizationHomepage := Some(url("https://cognite.com")),
-  version := "0.4.12",
+  version := "0.4.14",
```

Comment:
With this one we're on CDF v1, right? I think we should bump it to 0.5.x at least.

Reply:
Only for time series, and soft `v1` I guess since it doesn't actually expose `externalId` for time series. I suggest postponing the bump until next release which will include scala SDK and several resources to `v1`.